### DISCUSSION TITLE
learning benefits and features of weblate

### DISCUSSION THREAD

**Discussion by StanOsborne** [WeblateOrg/weblate#7835] [Ideas]
weblate website has lots of documentation but my questions are still unanswered.
github not providing obvious answers
1) no easy to find list of supported languages, or a search tool
2) documentation comparing weblate with other translation management tools"
3) list of links to projects using weblate; good examples

**Comment by nijel** [comment]
1. See https://docs.weblate.org/en/latest/admin/languages.html
2. Weblate is strong in version control integration, for the rest it might depend on personal preference and needs
3. You can find some at https://weblate.org/discover/

**Comment by StanOsborne** [comment]
This answers my questions and encourages my adoption of Weblate
with future FOSS projects.
